src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -1024,4 +997,60 @@ public Consumer setServiceType(String serviceType) {
         this.serviceType = serviceType;
         return this;
     }
+
+    public List<String> getEnvironmentIds() {
+        return this.environmentIds.entrySet().stream()
+            .sorted(Comparator.comparingInt(lhs -> Integer.parseInt(lhs.getKey())))
```

Comment:
Pedantic request: "lhs" is usually used to indicate the left-hand side of a comparison. Since we're isolated from that due to the handy comparator function, let's just call it "entry" or something to indicate that it's working with the elements from the entry set.